modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -2634,6 +2634,14 @@ public int compare(CodegenProperty one, CodegenProperty another) {
                 postProcessModelProperty(m, prop);
             }
         }
+        if(!m.isEnum
+                && m.anyOf.size()>1
+                && m.anyOf.contains("std::string")
```

Comment:
I think `std::string` is C++ only.

What about adding `x-is-string-enum-container` instead in the postProcessModel operation in the C++ pistache server generator?

Reply:
Thanks.

- Removed "isStringEnumContainer" and its setter from "DefaultCodeGen"
- added the setter to "CppPistacheServerCodegen.java" making it add a new entry to "vendorExtensions" as "x-is-string-enum-container"

Now it is pistache cpp server only. Making it global to all servers...; you may guide me..

Reply:
If other generators need the same, they can reuse the same code block in the postProcessModel method.